### PR TITLE
Document AMD FidelityFX Super Resolution 2.2 support

### PR DESCRIPTION
**Pull request by Calinou**
This documents its use both as an upscaling method and an antialiasing method.

~~**Marked as draft**, as comparison images need to be added. I'm running into a bug with mipmap bias not applying correctly in `master`, so I can't add them until I find a way to fix or work around the issue for the purpose of taking screenshots with https://github.com/Calinou/godot-antialiasing-comparison.~~
**Edit:** Fixed with the power of image editing. :slightly_smiling_face: 

Don't cherry-pick, as this feature is only available in Godot 4.2.

- See https://github.com/godotengine/godot/pull/81197.

### CODE REVIEW

**Comment by skyace65** [comment]
@Calinou is that mipmap issue still preventing you from getting screenshots?

**Comment by Calinou** [comment]
> @Calinou is that mipmap issue still preventing you from getting screenshots?

It still occurs in 4.2.beta3, but I worked around the issue by disabling 3D on one of the viewports and editing the resulting images to mash them together. It does the job :slightly_smiling_face:

File: tutorials/3d/3d_antialiasing.rst [hunk 1]
```diff
@@ -129,7 +162,7 @@ which is much more effective against specular aliasing. Nonetheless, exposing FX
 as an in-game option may still be worthwhile for players with low-end GPUs.
 
 FXAA introduces a moderate amount of blur when enabled (more than TAA when
-still, but less than TAA when the camera is moving).
+still, but less than TAA when the camera is moving).w
```
Comment (skyace65) [inline tutorials/3d/3d_antialiasing.rst:165]: I don't think you meant to add this

**Review by skyace65 — CHANGES_REQUESTED** [review]

File: tutorials/3d/3d_antialiasing.rst (hunk 1 above)
Comment (Calinou) [reply tutorials/3d/3d_antialiasing.rst]: Good catch, fixed :slightly_smiling_face:w

**Review by skyace65 — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

File: tutorials/3d/resolution_scaling.rst [hunk 2]
```diff
@@ -43,29 +46,44 @@ Scaling mode
...
   scaling.
+- **FSR 2.2:** AMD FidelityFX Super Resolution 2.2 (since Godot 4.2). Slowest,
+  but even higher quality compared to FSR1 and bilinear scaling. On slow GPUs,
+  the cost of FSR2 may be too expensive to be worth using it over bilinear
+  scaling or FSR1. To match FSR2 performance with FSR1, you need to use a lower
+  resolution scale factor.
 
 Here are comparison images between native resolution, bilinear scaling with 50%
-resolution scale and FSR 1.0 scaling with 50% resolution scale:
+resolution scale, FSR1 and FSR2 scaling with 50% resolution scale:
```
Comment (akien-mga) [inline tutorials/3d/resolution_scaling.rst:58]:
```suggestion
resolution scale, FSR1, and FSR2 scaling with 50% resolution scale:
```

**Review by mhilbrunner — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks!

**Pull request merged**